src/main/java/com/maxmind/minfraud/response/IpRiskReason.java

```diff
@@ -10,6 +10,11 @@ public final class IpRiskReason extends AbstractModel {
     private final String code;
     private final String reason;
 
+    /**
```

Comment:
You are still missing the description for this constructor:

```suggestion
    /**
     * Constructor for {@code IpRiskReason}. 
     *
``````